bip-0330.mediawiki

```diff
@@ -0,0 +1,299 @@
+<pre>
+  BIP: 330
+  Layer: Peer Services
+  Title: Transaction announcements reconciliation
+  Author: Gleb Naumenko <naumenko.gs@gmail.com>
+          Pieter Wuille <pieter.wuille@gmail.com>
+  Comments-Summary: ???
+  Comments-URI: ???
```

Comment:
First Comments-URI must be exactly "https://github.com/bitcoin/bips/wiki/Comments:BIP-0330" in bip-0330.mediawiki at scripts/buildtable.pl line 161, <$F> line 8.